mix.exs

```diff
@@ -21,6 +21,7 @@ defmodule Bottle.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
+      {:amqp, "~> 1.3"},
```

Comment:
It looks like the broadway adapter can also use 2.0 https://github.com/dashbitco/broadway_rabbitmq/blob/master/mix.exs#L32

We should update